src/riscv-hybrid-integration.adoc

```diff
@@ -88,6 +88,32 @@ CRE=1 for the current privilege level
 
 When the <<m_bit>> can be set follows the rules defined by <<ACPERM>>.
 
+[#m_bit_observe,reftext="Observing the CHERI Execution Mode"]
+==== Observing the CHERI Execution Mode
+
+The effective CHERI execution mode is given by the values of some CSRs and the
```

Comment:
Should probably add a xref to the CRE section here since "some" is quite vague?

Reply:
I did not add it because this is actually a subsection of the section that describes that and also because its already linked with the `<<m_bit>>` ref at the end of the sentence. The relevant text is 2 paragraphs before here: https://github.com/riscv/riscv-cheri/pull/314/files#diff-b366fcf81637594697f776b0c57bf43a576436df8b1fde7ed1ae593771025d94R82